### PR TITLE
Added better focus ui for arrow key / tab navigation

### PR DESCRIPTION
**Pull request by zainafzal08**
Updates the focus animation to be easier on the eyes when moving focus with the tab / arrow keys by disabling the ripple.

### CODE REVIEW

**Comment by E-T-** [comment]
> Updates the focus animation to be easier on the eyes when moving focus with the tab / arrow keys.
Wait, why is there a screencast link on Github? That seems strange, please remove it?

File: js/foam/ui/md/Halo.js [hunk 1]
```diff
@@ -215,7 +229,27 @@ CLASS({
         if (this.focusType === 'mouse') return;
 
         this.focusType = 'keyboard';
+        // Save all of the halo properties.
+        const save = {
+          animateGrowth: this.animateGrowth,
+          startAlpha: this.startAlpha,
+          easeInTime: this.easeInTime,
+          easeOutTime: this.easeOutTime
+        }
```
Comment (E-T-) [inline js/foam/ui/md/Halo.js]: semicolon here?

File: js/foam/ui/md/Halo.js [hunk 2]
```diff
@@ -151,11 +155,21 @@ CLASS({
...
+          this.r = Math.min(28, Math.min(this.$.clientWidth, this.parent.height)/2);
+        } else {
+          this.r = Math.max(28, Math.max(this.$.clientWidth, this.parent.height));
+        }
         this.alpha = this.startAlpha;
-        var recentering = this.recentering;
+        const recentering = this.recentering;
         this.X.animate(this.easeInTime, function() {
-          if ( recentering ) {
+          if ( !this.animateGrowth ) {
```
Comment (E-T-) [inline js/foam/ui/md/Halo.js]: Can you do

```
if (!this.animateGrowth) return;

if (recentering) {
...
} else {

}
```

**Review by E-T- — COMMENTED** [review]
I think the description of this patch should be updated :)

**Comment by zainafzal08** [comment]
> > Updates the focus animation to be easier on the eyes when moving focus with the tab / arrow keys.
> > Wait, why is there a screencast link on Github? That seems strange, please remove it?

Done.

File: js/foam/ui/md/Halo.js (hunk 1 above)
Comment (zainafzal08) [reply js/foam/ui/md/Halo.js]: Done :)

File: js/foam/ui/md/Halo.js (hunk 2 above)
Comment (zainafzal08) [reply js/foam/ui/md/Halo.js]: We still need line 179 which sets the alpha.

Comment (E-T-) [reply js/foam/ui/md/Halo.js]: Can you move line 179 above this?

**Review by E-T- — APPROVED** [review]

File: js/foam/ui/md/Halo.js (hunk 2 above)
Comment (zainafzal08) [reply js/foam/ui/md/Halo.js]: yep! Sorry i completely blanked there :p

**Pull request merged**